pkgs/development/python-modules/whisperx/default.nix

```diff
@@ -0,0 +1,75 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  setuptools,
+
+  ctranslate2,
+  faster-whisper,
+  nltk,
+  pandas,
+  pyannote-audio,
+  torch,
+  torchaudio,
+  transformers,
+
+  cudaSupport ? torch.cudaSupport,
+
+  pkgs,
+}:
+
+let
+  ctranslate = ctranslate2.override {
+    ctranslate2-cpp = pkgs.ctranslate2.override {
+      withCUDA = cudaSupport;
+      withCuDNN = cudaSupport;
+    };
+  };
+in
+buildPythonPackage rec {
+  pname = "whisperx";
+  version = "3.3.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "m-bain";
+    repo = "whisperX";
+    tag = "v${version}";
+    hash = "sha256-JJa8gUQjIcgJ5lug3ULGkHxkl66qnXkiUA3SwwUVpqk=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    ctranslate
+    faster-whisper
+    nltk
+    pandas
+    pyannote-audio # Missing from pyproject.toml, but used in `whisperx/vad.py`
+    torch
+    torchaudio
+    transformers
+  ];
+
+  # > Checking runtime dependencies for whisperx-3.3.2-py3-none-any.whl
+  # >   - faster-whisper==1.1.0 not satisfied by version 1.1.1
+  # This has been updated on main, so I expect this clause to be removed upon the next update.
+  pythonRelaxDeps = [ "faster-whisper" ];
+
+  pythonImportsCheck = [ "whisperx" ];
+
+  # No tests in repository
+  doCheck = false;
+
+  makeWrapperArgs = [ "--prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ pkgs.ffmpeg ]}" ];
```

Comment:
```suggestion
  makeWrapperArgs = [ "--prefix PATH : ${lib.getBin [ pkgs.ffmpeg ]}/bin" ];
```

Reply:
Why not `lib.makeBinPath [ pkgs.ffmpeg ]`?